project-base/docker/conf/docker-compose-win.yml.dist

```diff
@@ -98,15 +99,8 @@ services:
             - discovery.type=single-node
 
     microservice-product-search:
-        build:
-            context: ../microservices/product-search/docker
-            args:
-                www_data_uid: 82
-                www_data_gid: 82
+        image: shopsys/shopsys:microservice-test
```

Comment:
microservice test?

Reply:
I have answered this in the previous comment.